courses/machine_learning/deepdive2/structured/solutions/1_explore_babyweight.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<ul><li>valid_columns_set -&gt; columns_set (see comment above)</li><li>Remove the code comment directly above the function (You already have a docstring explaining the same thing)</li><li>In the docstring: </li><li class="ql-indent-1">statstics -&gt; statistics</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done</p>